contentstore/models.py

```diff
@@ -197,6 +198,7 @@ class Message(models.Model):
     )
     sequence_number = models.IntegerField(null=False, blank=False)
     lang = models.CharField(max_length=6, null=False, blank=False)
+    metadata = JSONField(default=dict)
```

Comment:
I think this should be added to the serializer, so that we can access this field in the API

Reply:
okay will do